roles/downstream-setup/tasks/main.yml

```diff
@@ -1,36 +1,14 @@
 ---
-- include: yum_repos.yml
-  when: yum_repos|length > 0 and
-        not cleanup
-  tags:
-    - yum-repos
-
-- name: Set remove_yum_repos on cleanup
-  set_fact:
-    remove_yum_repos: "[{% for repo in yum_repos %}'{{ repo.name}}',{% endfor %}]"
-  when: yum_repos|length > 0 and
-        cleanup and
-        remove_yum_repos|length == 0
-
-- include: remove_yum_repos.yml
-  when: remove_yum_repos|length > 0
-  tags:
-    - delete-yum-repos
-
-- include: disable_yum_repos.yml
-  when: disable_yum_repos|length > 0 and
-        not cleanup
-  tags:
-    - disable-yum-repos
-
-- name: Set enable_yum_repos on cleanup
-  set_fact:
-    enable_yum_repos: "{{ disable_yum_repos }}"
-  when: disable_yum_repos|length > 0 and
-        cleanup and
-        enable_yum_repos|length == 0
-
-- include: enable_yum_repos.yml
-  when: enable_yum_repos|length > 0
-  tags:
-    - enable-yum-repos
+# These are tasks where the variables used perform the action
+# defined by their name.  For example, `disable_yum_repos` would actually
```

Comment:
This feels like strange wording to me. Maybe something like "These are tasks which perform actions corresponding to the names of the variables they use" ?

The other paragraph could say "tasks which reverse the actions corresponding to..."